core/CHANGES.md

```diff
@@ -1,3 +1,18 @@
+# Copyright (c) 2015 YCSB contributors. All rights reserved.
```

Comment:
this markdown file still needs the license header changed over ot html comments.